templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -162,5 +162,16 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <extra_dir type="job_work" path="/data/jwd/main"/>
             <extra_dir type="temp" path="/data/jwd/tmp"/>
         </backend>
+
+        <backend type="swift" id="s3_cyfronet01" weight="0" store_by="uuid">
```

Comment:
is `type=swift` right?

Reply:
Yes, our ontology needs to be adopted. s3 refers in out current ontology to the AWS S3.

Reply:
OK, just making sure